packages/server/test/unit/modes/interactive_spec.js

```diff
@@ -27,13 +27,13 @@ describe('gui/interactive', () => {
   context('.getWindowArgs', () => {
     it('quits app when onClose is called', () => {
       electron.app.quit = sinon.stub()
-      interactiveMode.getWindowArgs({}).onClose()
+      interactiveMode.getWindowArgs(1234, {}).onClose()
```

Comment:
nit: getWindowArgs has a type of string for the url parameter. I don't think it impacts these tests, but you could replace with a string to head off any future confusion.

Reply:
4deada77e3419f44196f513396fa030218a979bc